setup.py

```diff
@@ -50,14 +50,15 @@
 ]
 
 install_requires = [
-    "Flask>=2.1.1,<2.2.0",
+    "Flask>=2.1.1,<2.3.0", # same upper pin as invenio/reana-server
     "Werkzeug>=2.1.0,<3.0.0",
     "gitpython>=2.1",
     "jsonpickle>=0.9.6",
-    "marshmallow>2.13.0,<=2.20.1",
+    "marshmallow>2.13.0,<3", # same upper pin as reana-server
     "packaging>=18.0",
     "reana-commons[kubernetes]>=0.95.0a2,<0.96.0",
-    "reana-db>=0.95.0a2,<0.96.0",
+    # commenting out so that I can test newer dependencies
+    # "reana-db>=0.95.0a2,<0.96.0",
```

Comment:
TODO: uncomment after https://github.com/reanahub/reana-db/pull/234 has been merged and reana-db has been released